apps/src/templates/sectionAssessments/CorrectAnswerIcon.story.jsx

```diff
@@ -6,9 +6,15 @@ export default storybook => {
     .storiesOf('SectionAssessments/CorrectAnswerIcon', module)
     .addStoryTable([
       {
-        name:'CorrectAnswerIcon',
+        name: 'CorrectAnswerIcon',
+        description: `
+            A green check mark is visible if students answer a question
```

Comment:
Actually, according to the [spec](https://docs.google.com/document/d/1-2-TAFVeYqIIXs1WKIgD3pNMqgUUhPZYwevFSWCHpoY/edit?ts=5a85fc0c#heading=h.sqqr8ieg1490) for the piece you are building, the green check mark will appear if the cell indicates the correct answer (the students can still get it right or wrong).

I'd also recommend re-naming your component to something more descriptive of what it does now, like `MultipleChoiceAnswerCell`, since it's not just the correct answer icon anymore. Now it's a component that represents the whole table cell.

Reply:
Thanks for the feedback.